modules/thisbook.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable */
```

Comment:
It is ok to sometimes disable some linters on some lines of code to create an allowance for your application to work. However, it is not a good practice to disable linters for a whole file when you have about 2 or 3 errors to fix. ♻️ 

- [ ] Kndly check and fix the errors being thrown by your linter checker as the main purpose for installing linters is to help your learning and make you are better developer. 👍🏿